utils/config/daos_agent.yml

```diff
@@ -17,12 +17,11 @@
 
 # Management server access points
 # Must have the same value for all agents and servers in a system.
-# default: hostname of this node at port 10001 for local testing
-#access_points: ['hostname1:10001','hostname2:10001','hostname3:10001']
-#access_points: [hostname1,hostname2,hostname3]
+# default: hostname of this node
+#access_points: ['hostname1']
 
 # Force different port number to connect to access points.
-# default: 10000
+# default: 10001
 #port: 10001
```

Comment:
I was hoping to remove this parameter altogether from the client/agent configs. IMO it doesn't really add any value and seems to be an ongoing point of confusion.

Reply:
as we discussed this will be done in a different PR

Reply:
https://jira.hpdd.intel.com/browse/DAOS-4017 the ticket that addresses the client side